### PR TITLE
Fix a bug causing resources with relative paths to load the wrong URL

### PR DESCRIPTION
**Pull request by imaustink**
We highjack clicks on `<a>` tags and use Ajax to load the content in. If we don't change the URL with `pushState` before we load the content into the DOM, all relative paths will be based off the previous page's URL. This causing the issue Justin reported in slack: https://bitovi.slack.com/archives/C09R50SAE/p1499289090158230

### CODE REVIEW

File: static/canjs.js [hunk 1]
```diff
@@ -270,8 +272,6 @@ function navigate(href) {
 			if(searchControl.searchResultsCache){
 				searchControl.renderSearchResults(searchControl.searchResultsCache);
 			}
-      
-      window.history.pushState(null, null, href);
 		},
 		error: function() {
 			window.history.pushState(null, null, href);
```
Comment (chasenlehara) [inline static/canjs.js]: Is this line no longer needed because we’re doing it before the Ajax request?

File: static/canjs.js [hunk 2]
```diff
@@ -218,6 +218,9 @@ function navigate(href) {
 			return xhr;
 		},
 		success: function(content) {
+			
+			window.history.pushState(null, null, href);
```
Comment (imaustink) [inline static/canjs.js]: @chasenlehara I think this is where we want it.

File: static/canjs.js [hunk 3]
```diff
@@ -25,6 +25,10 @@ var $articleContainer,
 	hasShownSearch;
 
 (function() {
+
+	// Google Analytics
+	ga('send', 'pageview', window.location.pathname);
```
Comment (imaustink) [inline static/canjs.js]: @chasenlehara don't we already send the initial page load here: https://github.com/canjs/bit-docs-html-canjs/blob/4610356cb1c509fa784cb3ae7b5ba871e92664ab/templates/layout.mustache#L29

Comment (chasenlehara) [inline static/canjs.js]: You are right, thanks for pointing that out! I’ve removed my commit.

**Pull request merged**